src/main/java/FibonacciEven.java

```diff
@@ -0,0 +1,19 @@
+
+import java.util.List;
+
+public class FibonacciEven {
+
+    public static List<Integer> generateOnlyEvenFibonacciNumbersSum(int limitFromUser){
+        int sum = 0;
+        long limit = limitFromUser;
+        for (Integer[] number = new Integer[]{0, 1}; ; number = new Integer[]{number[1], number[0] + (number[1])}) {
```

Comment:
Tę całą logikę da się zapisać prościej i czytelniej pomijając używanie tablicy. Postaraj się to przepisać z takiej formy na nieco prostszą i bardziej czytelną.